### PR TITLE
Bump autoenv to 1.0.0

### PR DESCRIPTION
**Pull request by pikeas**

### CODE REVIEW

File: Library/Formula/autoenv.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 class Autoenv < Formula
   homepage 'https://github.com/kennethreitz/autoenv'
-  url 'https://github.com/kennethreitz/autoenv/archive/v0.1.0.tar.gz'
-  sha1 '4d773ba8162b8f49abdfc07d0f8c956aa3353e0c'
+  url 'https://github.com/kennethreitz/autoenv/archive/c4bc3f5093f73e5c23563aa5cd66307e2804f35d.tar.gz'
```
Comment (mikemcquaid) [inline Library/Formula/autoenv.rb]: This needs to be a tagged, stable release.

Comment (pikeas) [inline Library/Formula/autoenv.rb]: Opened issue upstream about tagging a new release.

Comment (bfontaine) [inline Library/Formula/autoenv.rb]: Where did you see that this is the 1.0.0 version?

**Comment by dunn** [comment]
There's a tagged release on PyPi we can use: https://pypi.python.org/packages/source/a/autoenv/autoenv-1.0.0.tar.gz

**Comment by AnilRedshift** [comment]
PING :-) We use the AUTOENV_ENV_FILENAME and it isn't configured on the `brew install autoenv` version.

**Comment by ariscn** [comment]
@AnilRedshift You could try pinging the upstream autoenv issue as well.

**Comment by ariscn** [comment]
@mikemcquaid @dunn The PyPi release is 1.0.0, but is older than the changes in the autoenv repo. However, the repo did not update the version number.

Assuming that the repo owner remains unresponsive regarding bumping the version and tagging, is there any other way to get the latest changes for this project into homebrew?

**Comment by dunn** [comment]
> Assuming that the repo owner remains unresponsive regarding bumping the version and tagging, is there any other way to get the latest changes for this project into homebrew?

Unfortunately not; but there's a `head` resource, so `brew install --HEAD autoenv` will install the very latest version.

**Comment by bfontaine** [comment]
This PR needs to be rebased on top of `master`.

**Comment by bfontaine** [comment]
Ping?

**Comment by ariscn** [comment]
I'm a bit confused by your ping? Based on earlier comments, my understanding is that homebrew will only accept the PR if the project tags a new release, which they seem unwilling to do.

**Comment by bfontaine** [comment]
@ariscn Just wanted to see if there was anything new here but as you wrote it doesn’t seem so. Let’s close this for now. You can always install the latest code with `brew install --HEAD autoenv`; feel free to open another PR if upstream tags a new version but it seems unlikely given the repo activity. Thanks!

**Pull request closed without merging**